examples/with-firebase-cloud-messaging/static/firebase-messaging-sw.js

```diff
@@ -0,0 +1,8 @@
+importScripts('https://www.gstatic.com/firebasejs/4.8.1/firebase-app.js')
```

Comment:
Where does this function come from?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/importScripts